sdk/core/azure-json/src/main/java/com/azure/json/JsonProviders.java

```diff
@@ -14,7 +14,20 @@
 import java.util.ServiceLoader;
 
 /**
- * Handles loading an instance of {@link JsonProvider} found on the classpath.
+ * Utility class for {@link JsonProvider} that will the first implementation of {@link JsonProvider} found on the
+ * classpath to create instances of {@link JsonReader} or {@link JsonWriter}.
+ * <p>
+ * If no implementation of {@link JsonProvider} is found on the classpath a default implementation provided by this
+ * library will be used.
+ * <p>
+ * At this time, additional implementations of {@link JsonProvider} found on the classpath after the first will be
```

Comment:
This will be non-deterministic. Should we start with throwing an exception if more than one implementation is found? This allows us to add flexibility later.